### DISCUSSION TITLE
How to become a sponsor

### DISCUSSION THREAD

**Discussion by hooray** [element-plus/element-plus#4611] [Q&A] [answered]
我在 ReadMe 里有看到赞助链接，但我不清楚如果要在官网首页的赞助商列表里出现，需要怎么样的赞助条件，是否有官方成员可以回答下？

[a comment by an automated account is not shown]

**Comment by 0song** [comment, marked as answer]
Hi, feel free to contact us via element-plus@outlook.com.

Cheers.
